lib/elixir/pages/Guards.md

```diff
@@ -70,7 +71,7 @@ Other constructs that support guards are [`for`](`for/1`), [`with`](`with/1`), [
 
 ## Failing guards
 
-A function clause will only be executed if and only if its guard expression evaluates to `true`. If any other value is returned, the function clause will be skipped. In particular, guards have no concept of "truthy" or "falsey".
+As mentioned before, a function clause will be executed if and only if its guard expression evaluates to `true`. If any other value is returned, the function clause will be skipped. In particular, guards have no concept of "truthy" or "falsey". In a nutshell, when it comes to guards, *it's `true` or nothing*.
```

Comment:
"As mentioned before" is usually an anti-pattern.

1. Or the information is worth being repeated, which means the "as mentioned before" is irrelevant
2. Or the information is not relevant, which means it should not be repeated at all

Reply:
I am not sure adding "In a nutshell, when it comes to guards, *it's `true` or nothing*." helps. At this point, we are saying the same thing 4 times:

  1. if and only if its guard expression evaluates to `true`
  2. If any other value is returned, the function clause will be skipped (same thing, but inverted)
  3. In particular, guards have no concept of "truthy" or "falsey" (can be derived from 1)
  4. In a nutshell, when it comes to guards, *it's `true` or nothing*. (same as 1)

Please remove. :)

Reply:
I reverted the changes, but I removed a redundant "only"